README.md

```diff
@@ -9,7 +9,7 @@ You must put your name and roll number as comment in the pull request to make su
 There are 4 types of pull requests anyone contributing to the code should submit.
 The type of pull request will be mentioned by the reviewers.
 
-### Tyep 1:
+### Type 1:
```

Comment:
This is type-1 but other PRs have done this. Look out for something that hasn't yet been spotted. You can fix grammar or whitespaces as well. Good Luck.